Classes/WPEditorView.m

```diff
@@ -368,11 +368,63 @@ - (void)stopObservingKeyboardNotifications
 
 - (void)keyboardDidShow:(NSNotification *)notification
 {
+    BOOL isiOS7OrEarlier = ([[[UIDevice currentDevice] systemVersion] floatValue] < 8.0);
```

Comment:
Good call.  I ended up adding a method to `WPDeviceIdentification` since that class has the intention of providing these methods.